spec/lib/fat_zebra/purchase_spec.rb

```diff
@@ -65,11 +74,21 @@
 
   describe '#capture', :vcr do
     let(:purchase) { FatZebra::Purchase.create(valid_purchase_payload.merge(capture: false)) }
-    subject(:capture) { purchase.capture }
 
-    it { is_expected.to be_accepted }
-    it { is_expected.to be_successful }
-    it { expect(capture).to be_a(FatZebra::Purchase) }
+    context 'success' do
+      subject(:capture) { purchase.capture }
+
+      it { is_expected.to be_accepted }
+      it { is_expected.to be_successful }
+      it { expect(capture).to be_a(FatZebra::Purchase) }
+    end
+
+    context '200 error' do
```

Comment:
should it be 400 error, but really the context is that the amount is too big.. the http response code is irrelevant are we are not operating at http level here